libbeat/kibana/index.go

```diff
@@ -0,0 +1,120 @@
+package kibana
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/elastic/beats/libbeat/common"
+)
+
+type Index struct {
+	Version   string
+	IndexName string
+	BeatDir   string
+	BeatName  string
+
+	fieldsYaml       string
+	targetDirDefault string
+	targetDir5x      string
+	targetFilename   string
+}
+
+func (i *Index) Create() ([]string, error) {
```

Comment:
I assume CI is going to complain that there are no comments here.

Reply:
hm, interestingly I haven't seen it recently, so it seems it's not executed anymore.